<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsInfoView</class>
 <widget class="QWidget" name="OsInfoView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblCategory">
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblCategoryValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblVersion">
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lblVersionValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblNameValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
